summit.html

```diff
@@ -48,12 +48,381 @@ <h1 class="section-heading">The practicalities</h1>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>9:15</td>
+            </tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Eiffel for beginners</summary>
+                  <div>
+                    <p>
+                      Why would you want to use events for expressing
+                      what goes on in your CI/CD pipeline, and what
+                      real-world problems can Eiffel help out with?
+                    </p>
+                    <p>
+                      <b>Speaker:</b> TBD (Axis &amp; Ericsson)
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>9:30</td>
+            </tr>
+
+            <tr><td><i>Break</i></td><td></td><td>10:00</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Eiffel for beginners, continued</summary>
+                  <div>
+                    <p>
+                      We continue the previous session after the break.
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>10:30</td>
+            </tr>
+
+            <tr><td><i>Lunch</i></td><td></td><td>11:30</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Eiffel at Volvo</summary>
+                  <div>
+                    <ul>
+                      <li>Eiffel traceability enabler (VSF Product
+                        Metrics)</li>
+                      <li>Eiffel for triggering (VSF CI)</li>
+                      <li>Eiffel as a bridge between companies (Volvo IT)</li>
+                    </ul>
+                    <p>
+                      <b>Speakers:</b> TBD
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>12:30</td>
+            </tr>
+
+            <tr><td><i>Break</i></td><td></td><td>14:00</td></tr>
+
+            <tr>
+              <td>
+                <details>
+                  <summary>Signing of Eiffel events</summary>
+                  <div>
+                    <p>
+                      Presentation of how you can use Eiffel event
+                      signing to uphold the integrity of events and
+                      authenticate their sender.
+                    </p>
+                    <p>
+                      <b>Speakers:</b> Magnus Bäck
+                    </p>
+                  </div>
+                </details>
+              </td>
+              <td><span class="tag bd-success">Beginner</span></td>
+              <td>15:00</td>
```

Comment:
Maybe this should be 14.30? Currently two sessions start at 15. Are they in parallel?

Reply:
Yes it shall be 1430

Reply:
Yeah, several times were off. Update pushed.